HOWTO.md

```diff
@@ -212,6 +215,61 @@ Use `fdo-owner-tool dump-ownership-voucher`:
 fdo-owner-tool dump-ownership-voucher your_ownership_voucher --outform cose > your_ownership_voucher.cose
 ```
 
+### How to export OVs from the Manufacturer Server (Database specific)
+
+Use `fdo-owner-tool export-manufacturer-vouchers`:
+
+```
+$ fdo-owner-tool export-manufacturer-vouchers --help
+Exports a single or all the ownership vouchers present in the Manufacturer DB
+
+Usage: fdo-owner-tool export-manufacturer-vouchers <DB_TYPE> <DB_URL> <PATH> [GUID]
+
+Arguments:
+  <DB_TYPE>  Type of the Manufacturer DB holding the OVs [possible values: sqlite, postgres]
+  <DB_URL>   DB connection URL, or path to the DB file
+  <PATH>     Path to dir where the OVs will be exported
+  [GUID]     GUID of the voucher to be exported, if no GUID is given all the OVs will be exported
+```
+
+For example:
+
+```bash
+fdo-owner-tool export-manufacturer-vouchers postgres \
+postgresql://test:test@localhost/test_manufacturer \
+/path/to/manufacturer-exports/
+```
```

Comment:
I added the export-import tools to the owner tool since the owner is the one that owns the OVs, and similar methods to handle OVs and device credentials are in the owner tool CLI, not in the admin tool.
The admin tool is only used to generate certs and run the aio tool, it does not have any OV handling capabilities.